packages/developer-extension/src/background/domain/replaceBundles.ts

```diff
@@ -0,0 +1,51 @@
+import { listenAction } from '../actions'
+import { DEV_LOGS_URL, DEV_RUM_RECORDER_URL, DEV_RUM_URL } from '../constants'
+import { setStore, store } from '../store'
+
+chrome.webRequest.onBeforeRequest.addListener(
+  (info) => {
+    if (store.useDevBundles) {
+      const url = new URL(info.url)
+      if (url.pathname.includes('logs')) {
+        return { redirectUrl: DEV_LOGS_URL }
+      }
+      if (url.pathname.includes('rum')) {
+        return {
+          redirectUrl: store.useRumRecorder ? DEV_RUM_RECORDER_URL : DEV_RUM_URL,
+        }
+      }
+    } else if (store.useRumRecorder && /\/datadog-rum(?!-recorder)\.js$/.test(info.url)) {
+      return {
+        redirectUrl: info.url.replace(/\.js$/, '-recorder.js'),
+      }
+    }
+    return
+  },
+  {
+    types: ['script'],
+    urls: [
+      // TODO: implement a configuration page to add more URLs in this list.
+      'https://www.datadoghq-browser-agent.com/datadog-logs.js',
+      'https://www.datadoghq-browser-agent.com/datadog-rum.js',
+      'https://www.datadoghq-browser-agent.com/datadog-rum-recorder.js',
```

Comment:
What if the SDK is configured to use a different intake domain? Is it possible generate these URLs based on the actual RUM and logs configurations?

Reply:
This list is not intake domains, but I agree it would be great to be able to configure bundle URLs and intake lists. I'll add a list of feature ideas somewhere.

I'm not sure how to discover such URLs based on page the RUM and Logs configurations though.